CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Unreleased
+
+### Adds
+
+* Use `mergeWithCustomize` when merging extended source Webpack configuration. Introduce overideable asset module methods `srcCustomizeArray` and `srcCustomizeObject`, with reasonable default behavior, for fine tuning Webpack config arrays and objects merging. More info - [the Webpack mergeWithCustomize docs](https://github.com/survivejs/webpack-merge#mergewithcustomize-customizearray-customizeobject-configuration--configuration)
+
+### Fixes
+
+* The module `webpack.extensions` configuration is not applied to the core Admin UI build anymore.
```

Comment:
👍 Please clarify that this was never intended to be supported, as the "apos" build is intentionally separate to avoid ever breaking the expectations of official admin UI modules.